web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v370/migrate-default.sql

```diff
@@ -13,6 +13,9 @@ INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('metada
 INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('metadata/pdfReport/footerRight', '{date}', 0, 12507, 'y');
 INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('metadata/pdfReport/pdfName', 'metadata_{datetime}.pdf', 0, 12507, 'n');
 
+-- Update GML namespace for moving from ISO19139:2005 to ISO19139:2007
+UPDATE Metadata SET data = replace(data, '"http://www.opengis.net/gml"', '"http://www.opengis.net/gml/3.2"') WHERE data LIKE '%"http://www.opengis.net/gml"%' AND schema = 'iso19139';
```

Comment:
What about records that have `schemaLocation` defined to use `2005` version (not relaying on `schema-ident.xml` to add the `schemaLocation`?

Reply:
We can but it is not usually a good practice to set schemaLocation. It is better to let the system do it. We are not providing template with schemaLocation (except the fragment ones) so we should not have much record with it defined. 

Will add a replace for it.

Reply:
Added a SQL statement for it.